eclipse-scout-core/src/form/fields/datefield/DateField.js

```diff
@@ -1385,10 +1385,16 @@ export default class DateField extends ValueField {
     this.popup = this.createDatePopup();
     this.popup.open();
     this.$dateField.addClass('focused');
-    this.popup.on('remove', event => {
-      this._onPopupRemove(event);
-      this.popup = null;
+    // After picking a date the popup gets closed and destroyed. The method Widget#destroy triggers a 'remove' event
```

Comment:
I don't think the comment here is necessary, it is described in the commit message well enough. Destroy is the better event to listen for, so the chance that it will be reverted to remove is very small.